opm/simulators/flow/ExtraConvergenceOutputThread.cpp

```diff
@@ -112,6 +112,47 @@ namespace {
         return maxChar;
     }
 
+
+    std::string severityToString(const Opm::ConvergenceReport::Severity s)
+    {
+        using S = Opm::ConvergenceReport::Severity;
+        switch (s) {
+            case S::None:
```

Comment:
Is there a reason the `case` labels are indented here and not in the `wellFailure` helper function?